test/CallbackStreamTest.php

```diff
@@ -89,13 +89,13 @@ public function testGetSize()
         $this->assertNull($ret);
     }
 
-    /**
-     * @expectedException RuntimeException
-     */
     public function testTell()
     {
         $stream = new CallbackStream(function () {
         });
+
+        $this->setExpectedException(RuntimeException::class);
```

Comment:
use `expectException()` instead, as `setExpectedException()` is deprecated and will be removed in phpunit 6